oceannavigator/frontend/src/components/LineWindow.jsx

```diff
@@ -65,29 +64,6 @@ class LineWindow extends React.Component {
     this._mounted = false;
   }
 
-  componentWillReceiveProps(props) {
```

Comment:
`componentWillReceiveProps` is deprecated in newer React versions so this needs to be removed. Oddly enough deleting this had no noticeable difference on functionality :shrug: